### DISCUSSION TITLE
More information in navigation panel

### DISCUSSION THREAD

**Discussion by Esmolante** [organicmaps/organicmaps#3981] [Ideas]
I appreciate the cleanliness of OM's navigation mode. However, I can think of a few improvements:
In the line under the speed (where it is now km/h) could be displayed the altitude, which is useful in regions with mountainous terrain. 
In the line under the distance to destination (where it is now km), could be displayed the distance to the next stage .
As an option, a thin ribbon could be included between the numbers and the map, a compass. 
As an example of the compass, here is a screenshot of the old iGo 8:
https://http2.mlstatic.com/software-de-navegacion-igo-8-para-gps-y-auto-estreos-D_NQ_NP_439601-MLM20375496121_082015-F.jpg
I suggest each 90º sector is displayed in different colours.

**Comment by biodranik** [comment]
Can you please mention it in #1104? The tricky part is to leave the screen uncluttered. There is also related #3745

**Comment by Esmolante** [comment]
Of course I'll do. 
I appreciate the clean view of OM, but I think there's room to add what I suggest. Next stage, in particular, could be necessary in many cases.
I'm thinking in something like that:
![Organic Maps1](https://user-images.githubusercontent.com/109362590/204381089-26369266-d183-4401-bef2-7d35111bbfda.jpg)

And, with compass (under an option, because I know that is not usefull for everybody).

![Organic Maps2](https://user-images.githubusercontent.com/109362590/204381340-4a3ca2e7-9478-4804-8aa7-88bae69b0775.jpg)

**Reply by helmutundarnold** [reply]
Would be nice if the information can be configured like in the Navigator-App.
